src/Variable.js

```diff
@@ -1,18 +1,49 @@
 import React from "react";
+import { LineChart } from 'react-d3';
+
 import "./Variable.css";
 
-const Variable = ({ value, onChange, name }) => (
+const Variable = ({ value, onChange, name, result }) => (
   <div className="Variable">
     <label>
       {name}:
-      {" "}
+      <br />
+      <LineChart
+        legend={false}
+        data={calculateSeries(value, result)}
+        width={600}
+        height={200}
+      />
+      <input
+        type="range"
+        value={value}
+        min={1}
+        max={30}
+        onChange={e => onChange(name, e.target.value)}
+      />
       <input
         type="number"
         value={value}
+        min={1}
+        max={30}
         onChange={e => onChange(name, e.target.value)}
       />
     </label>
   </div>
 );
 
+let calculateSeries = (value, result) => {
+  var lineData = [
+    {
+      name: 'series1',
+      values: []
+    }
+  ]
+  for(var i =1; i<=30; i++) {
+    lineData[0].values.push({x: i, y: (i/value) * result});
```

Comment:
This is not actually taking into account the actual equation but is instead just creating a linear graph scaled by the result.

Reply:
Yep. As we have parameters that aren't linear, we can describe how they affect the end result and change the computation based off of that.  Computing just the single variables effect instead of recomputing the entire equation will likely result in better performance. Ideally the variables will have a value and some description of their effect on the result.